src/nimblepkg/tools.nim

```diff
@@ -164,3 +164,12 @@ proc getNimbleUserTempDir*(): string =
   else:
     tmpdir = getTempDir()
   return tmpdir
+
+proc getSymlinkFilename*(filename: string, options: Options): string =
+  let spl = filename.splitFile
+  result = options.getBinDir() / (block:
+    if spl.ext == ".out":
+      spl.name
+    else:
+      spl.name.addFileExt(spl.ext)
+  )
```

Comment:
Doesn't this work without the `block`?

Reply:
if not just refactor the `if` into a variable above this line.

Reply:
Done.